_includes/footer.html

```diff
@@ -1,30 +1,169 @@
 <footer class="usa-footer site-footer" role="contentinfo">
-  <div class="footer-section-bottom bg-base-lighter">
-    <div class="grid-container">
-      <div class="grid-row padding-3">
-        <div class="tablet:grid-col">
-          <div class="logo-links">
-            <a class="footer-logo media_link" href="https://cloud.gov">
-              {% image_with_class "./_img/logo-img.png" "width-10 height-auto" "Agency logo" %}
-            </a>
-          </div>
+    <div class="footer-section-bottom usa-footer__primary-section bg-accent-warm-light border-top border-accent-warm-dark">
+        <div class="grid-container  padding-bottom-4">
+            <div class="grid-row grid-gap-4">
+                <div class="mobile-lg:grid-col-6 desktop:grid-col-3">
+                    <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
+                        <h4 class="usa-footer__primary-link">Get in touch</h4>
+                        <ul class="usa-list usa-list--unstyled">
+                            <li class="usa-footer__secondary-link"><a href="/contact">Customer support</a></li>
```

Comment:
For any relative links on the site, you will want to use [11ty's url filter](https://www.11ty.dev/docs/filters/url/) to properly append a possible path prefix. For all of Pages site's preview deployments, the preview is at a subpath and this url filter will automatically append it to the path. We feed the path value at a site's preview build time. 

```suggestion
                            <li class="usa-footer__secondary-link"><a href="{{ '/contact' | url }}">Customer support</a></li>
```

Look for any other relative links not using the `url` filter and update them to use it.

Reply:
Updating any of the template or includes files now will be pick up in all of the following PR's once you rebase them.